packages/client/src/Redirect.re

```diff
@@ -0,0 +1,37 @@
+let component = ReasonReact.statelessComponent("Redirect");
+
+// @todo: fix this duplicated ligic(#rederectings)
```

Comment:
Do you have an idea when you are going to do this? Can you make a story for it?

Reply:
Here I need to rethink the application structure more before doing it. For now I do not properly imagine how to do it. 

But I have added new issue with steps: 
https://github.com/OAGr/foretold/issues/434